scripts/vendor.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+if [[ -z "$MOD_PATH" || -z "$MOD_VERSION" ]]; then
```

Comment:
So this needs two env vars set

- MOD_PATH (e.g. github.com/docker/buildx)
- MOD_VERSION (tag / branch / commit)

Reply:
I think this should be optional, and done directly in the bake definition